src/modules/hunting/secrets.py

```diff
@@ -0,0 +1,45 @@
+import json
+import logging
+import os
+
+
+import requests
+
+from ...core.events import handler
+from ...core.events.types import Vulnerability, Event, OpenPortEvent
+from ...core.types import  Hunter, KubernetesCluster, AccessRisk
+
+
+""" Vulnerabilities """
+class secretsAccess(Vulnerability, Event):
+    """ Accessing the pod's secrets within a compromised pod might disclose valuable data to a potential attacker"""
+
+    def __init__(self, evidence):
+        Vulnerability.__init__(self, KubernetesCluster, name="Accessed to pod's secrets", category=AccessRisk)
+        self.evidence = evidence
+
+# Passive Hunter
+#should change the subscribtion here... (openPortEvent isnt relevant..)
```

Comment:
You could do this whenever you find you're running as a pod. Maybe under HostScanEvent, if config.pod is true we should publish a new event called, say, RunningAsPodEvent and use that to trigger this secrets test wdyt?

Reply:
Yea I was going to change it to the HostScanEvent, seems like a great idea to add a RunningAsPodEvent.
Hopefully we won't face that race condition again when subscribing to an event that was thrown from another event..